pkg/detectors/aha/aha.go

```diff
@@ -74,6 +83,30 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return results, nil
 }
 
+func verifyAha(ctx context.Context, client *http.Client, resMatch, resURLMatch string) (bool, error) {
+	url := fmt.Sprintf("https://%s/api/v1/me", resURLMatch)
+	req, err := http.NewRequestWithContext(ctx, "GET", url, nil)
+	if err != nil {
+		return false, err
+	}
+	req.Header.Add("Accept", "application/vnd.aha+json; version=3")
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", resMatch))
+	res, err := client.Do(req)
+	if err != nil {
+		return false, err
+	}
+	defer res.Body.Close()
+
+	// https://www.aha.io/api
+	if res.StatusCode == http.StatusOK {
+		return true, nil
+	} else if res.StatusCode != http.StatusUnauthorized && res.StatusCode != http.StatusNotFound {
```

Comment:
i didn't realize i'd be asking the same question five times until i was already in too deep :(

Reply:
also manually checked